internal/server/server.go

```diff
@@ -164,6 +171,30 @@ func errorHandler(logger Logger, tmplError template.Template, prefix, siriusURL
 					correlationId = statusError.CorrelationId
 				}
 
+				if strings.Contains(r.Header.Get("Accept"), "application/json") {
```

Comment:
This could be wrong, if I sent up `Accept: text/html, application/json;q=0.8` I'd get JSON but want HTML. Having said that, I'm not sure if we care?

Reply:
Personally I don't think we care enough to do full parsing, but I've simplified to an exact match to avoid that case (and since we're in full control of what header we send).